python-client/docs/registration.md

```diff
@@ -86,7 +86,7 @@ When complete, this command returns a new version ID for the project.
 
 In-flight function invocations will continue to run using the old version of the function definitions,
 but future function invocations will use the new version of the code (unless a historical version ID
-is explicitly specified in the invocation call).
+is [explicitly specified](./invocations.md#invoking-functions-from-old-project-versions) when making the invocation).
```

Comment:
The original was wrong - the versionId is not actually passed to the `function.start()` method.

Also, a link is helpful

Reply:
But that doesn't change much. I understand "making the invocation" the same as "calling the function". And as you noted versionId is not actually passed when calling the function. 

"unless a historical version ID is explicitly specified in the relevant `get_deployed_function` function call"